src/mono/wasm/templates/Microsoft.NET.Runtime.WebAssembly.Templates.csproj

```diff
@@ -24,10 +24,5 @@
 
   <Target Name="CreateManifestResourceNames" />
   <Target Name="CoreCompile" />
-  <Target Name="_SetProductVersion" DependsOnTargets="GetProductVersions" BeforeTargets="Pack">
-    <PropertyGroup>
-      <PackageVersion>$(ProductVersion)</PackageVersion>
-    </PropertyGroup>
-  </Target>
```

Comment:
From what I could see, this setting shouldn't be necessary as PackageVersion is already automatically set by Arcade.